wgpu-hal/src/gles/mod.rs

```diff
@@ -113,13 +115,26 @@ bitflags::bitflags! {
     /// change the exposed feature set.
     struct PrivateCapabilities: u32 {
         /// Support explicit layouts in shader.
-        const SHADER_BINDING_LAYOUT = 0x0001;
+        const SHADER_BINDING_LAYOUT = 1 << 0;
         /// Support extended shadow sampling instructions.
-        const SHADER_TEXTURE_SHADOW_LOD = 0x0002;
+        const SHADER_TEXTURE_SHADOW_LOD = 1 << 1;
         /// Support memory barriers.
-        const MEMORY_BARRIERS = 0x0004;
+        const MEMORY_BARRIERS = 1 << 2;
         /// Vertex buffer layouts separate from the data.
-        const VERTEX_BUFFER_LAYOUT = 0x0008;
+        const VERTEX_BUFFER_LAYOUT = 1 << 3;
+    }
+}
+
+bitflags::bitflags! {
+    /// Flags that indicate necessary workarounds for specific devices or driver bugs
+    struct Workarounds: u32 {
+        // Needs workaround for Intel Mesa bug:
+        // https://gitlab.freedesktop.org/mesa/mesa/-/issues/2565.
+        //
+        // This comment
+        // (https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/4972/diffs?diff_id=75888#22f5d1004713c9bbf857988c7efb81631ab88f99_323_327)
+        // seems to indicate all skylake models are effected.
+        const WORKAROUND_MESA_I915_SRGB_SHADER_CLEAR = 1 << 0;
```

Comment:
Now that this is in a workarounds struct, the individual const shouldn't be prefixed with WORKAROUND

Reply:
That makes sense. Fixed!